network-store-server/src/main/java/com/powsybl/network/store/server/QueryCatalog.java

```diff
@@ -263,10 +263,12 @@ public static String buildInsertTemporaryLimitsQuery() {
                 " values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)";
     }
 
-    public static String buildDeleteTemporaryLimitsVariantEquipmentQuery() {
+    public static String buildDeleteTemporaryLimitsVariantEquipmentINQuery(int numberOfValues) {
         return "delete from temporarylimit where " +
```

Comment:
I'd put a check here like this :
`if (numberOfValues < 1) {
            throw new IllegalArgumentException("Function should not be called without values to insert.");
        }`

Reply:
done